app/views/partials/error_message_form_input.scala.html

```diff
@@ -27,7 +27,7 @@
 
 @if(form.errors(errorKey).nonEmpty) {
   @for(error <- form.errors(errorKey)) {
-    <span id="@error.message-errorKey" class="error-message">
+    <span id="manage-errorKey" class="error-message">
```

Comment:
This looks like it changes this partial so that it is no longer for general use throughout the application. 
If it's used in several pages it can have unintended effects on other pages.
If it is used in just one page, please inline it into that one page and delete this partial.

Reply:
Also, if there are multiple errors, this element id is not unique